services/worker-manager/src/provider.js

```diff
@@ -6,42 +9,53 @@ class Provider {
    * a cloud provider for terminating/listing instances. Any provisioning
    * logic should be started in `initiate` below.
    */
-  constructor({id, monitor, notify}) {
-    this.id = id;
+  constructor({
```

Comment:
so I noticed in `provisioner.js`, line 103, we use `provider.prepare` but I don't see this method here?

Reply:
It is unneeded for this provider currently.